hw3/01-characters.js

```diff
@@ -1,4 +1,55 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
 
-fetch(url);
+//reference --https://www.javascripttutorial.net/javascript-fetch-api/
+
+let section = document.querySelector("section");
+
+function show(character) {
+  let div = document.createElement("div");
+  let image = document.createElement("img");
+  let name = document.createElement("h2");
+  let title = document.createElement("p");
+
+  div.className = "image-card";
+
+  image.className = "image-style";
+  image.src = character.imageUrl;
+  image.alt = character.image;
```

Comment:
Use a descriptive alternative text, not just the image file name.

![ssalttext](https://github.com/ssiddhi/frontend/assets/38127011/9d543b44-68ed-4ac4-907d-72d01d0c77da)